packages/react-tabs/src/components/Pivot/index.ts

```diff
@@ -1,5 +1 @@
-export * from './Pivot';
-export * from './Pivot.base';
-export { PivotItem } from './PivotItem';
-export * from './Pivot.types';
-export * from './PivotItem.types';
+export * from '../Tabs/index';
```

Comment:
Is this file still needed? Couldn't you just have the `Pivot.ts` file under `src` refer to `Tabs/index` instead of keeping this file around? That way we could remove the `Pivot` folder completely.

Reply:
I was planning to use this file in Part 2 to re-export the Pivot, etc. names (as aliases of Tabs, etc.). Although I suppose I can just use react-tabs/src/Pivot.ts to do that...